src/sentry/integrations/vercel/redirect.py

```diff
@@ -0,0 +1,24 @@
+from django.http import HttpResponseRedirect
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.base import Endpoint
+from sentry.models import Integration
+from sentry.utils.http import absolute_uri
+
+
+class VercelConfigurationRedirect(Endpoint):
+
+    authentication_classes = ()
+    permission_classes = ()
+    provider = "vercel"
+
+    def get(self, request: Request) -> Response:
```

Comment:
I might be misunderstanding what's going on here, but can't this be accomplished with the static URL links feature that Steve shipped a while back?

I think if you make the configuration URL on Vercel something like:
```
https://sentry.io/orgredirect/organizations/__orgSlug__/integrations/vercel/
```
It should just work.